migrator/migrator-servlet/src/main/java/com/google/phonenumbers/ServletMain.java

```diff
@@ -43,6 +43,16 @@
 public class ServletMain extends HttpServlet {
 
   public static final int MAX_UPLOAD_SIZE = 50000;
+  /**
+   * Countries with large valid number ranges cannot be migrated using the web application due to request timeouts. The
+   * command line tool must be used in such cases.
```

Comment:
Maybe add a TODO here (for someone else in the future) of changing the request to have a longer timeout and show a spinner in the UI while the user is waiting?